source/decisions/2023-10-10-appoint-daniele-procida-as-trustee.rst

```diff
@@ -0,0 +1,8 @@
+2023-10-10 Appoint Daniele Procida as Trustee
+=============================================
+
+Resolution
+----------
+
+Daniele Procida was appointed by the trustees as a trustee of the UKPA in accordance
```

Comment:
Non-blocking suggestion, but I think "is" makes more sense than "was" in a decision document?

(I wonder if it's time we grew a style guide for these!)

Reply:
Actually, I think this needs to be worded as a resolution much like https://ukpa-internaldocs.readthedocs.io/en/latest/decisions/2020-11-13-membership-extension/

```suggestion
To appoint Daniele Procida  a trustee of the UKPA in accordance
```

Reply:
Thanks, done